contributing.md

```diff
@@ -1,14 +1,51 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
 
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+# Contributing to Network @ Linux System Roles
```

Comment:
```suggestion
# Contributing to the Network Linux System Role
```

Reply:
I wasn't sure if I could write it like the second one, thanks!